nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,85 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+reviewed: 2016-03-22
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
+
+
+## How it works
+
+The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues while the queue implementation logic is executed inside the client process.
+
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Throughput on par with MSMQ
+ * Scaling out of processing is simple because of support for competing consumers at the queue level
+
+## Usage scenarios - deployment size
+
+### Tiny
+
+The SQL Server transport is an ideal choice for extending an existing web application with asynchronous processing capabilities as an alternative for tedious batch jobs that tend to quickly get out of synch with the main codebase. Assuming the application already uses SQL Server as a database, this scenario does not require any additional infrastructure.
+
+The queue tables are hosted in the same catalog as business data and the NServiceBus runtime can be hosted in the web worker. In some cases there is a need for a separate process for NServiceBus. In most cases these systems consists of 1 to 3 non-scaled out endpoints. Because they consist of a single logical service or bounded context, there is usually no need to create separate schemas for the queues.
+
+### Small
+
+The SQL Server transport is a good choice for a pilot project to prove feasibility of NServiceBus in a given organization as well as for a small, well-defined green field application. It usually requires nothing more than a database. Such a system usually consists of 2-10 fairly independent non-scaled out endpoints.
+
+The best option is to store the queues in the same catalog as the business data. Schemas can be used to make maintenance easier and allow for finer-grained security. 
```

Comment:
Not really, IMHO I think Outbox is too restrictive the way it works, it would mean all business data needs to live in a single database which in practice is never true.
So I am saying that unfortunately DTC is the way to go.